plugins/woocommerce/client/legacy/js/frontend/order-attribution.js

```diff
@@ -31,34 +57,27 @@
 		const setFields = () => {
 
 			if ( sbjs.get ) {
-				$( `input[name="${prefix}type"]` ).val( sbjs.get.current.typ );
-				$( `input[name="${prefix}url"]` ).val( sbjs.get.current_add.rf );
-
-				$( `input[name="${prefix}utm_campaign"]` ).val( sbjs.get.current.cmp );
-				$( `input[name="${prefix}utm_source"]` ).val( sbjs.get.current.src );
-				$( `input[name="${prefix}utm_medium"]` ).val( sbjs.get.current.mdm );
-				$( `input[name="${prefix}utm_content"]` ).val( sbjs.get.current.cnt );
-				$( `input[name="${prefix}utm_id"]` ).val( sbjs.get.current.id );
-				$( `input[name="${prefix}utm_term"]` ).val( sbjs.get.current.trm );
-
-				$( `input[name="${prefix}session_entry"]` ).val( sbjs.get.current_add.ep );
-				$( `input[name="${prefix}session_start_time"]` ).val( sbjs.get.current_add.fd );
-				$( `input[name="${prefix}session_pages"]` ).val( sbjs.get.session.pgs );
-				$( `input[name="${prefix}session_count"]` ).val( sbjs.get.udata.vst );
-
-				$( `input[name="${prefix}user_agent"]` ).val( sbjs.get.udata.uag );
+				for( const [ key, value ] of Object.entries( wc_order_attribution.sbjsDataToSchema( sbjs.get ) ) ) {
+					$( `input[name="${prefix}${key}"]` ).value = value;
+				}
 			}
 		};
 
 		/**
-		 * Add source values to checkout.
+		 * Add source values to the classic checkout.
 		 */
-		$( document.body ).on( 'init_checkout', () => { setFields(); } );
+		if ( $( '.woocommerce form.checkout' ) !== null ) {
```

Comment:
This doesn't work for classic checkout, there's no form element that matches this selector. This seems to work:
```suggestion
		if ( $( 'form.woocommerce-checkout'' ) !== null ) {
```

Reply:
Sorry, it works with `/shortcode → [woocommerce_checkout]` but not with `/Classic Checkout`:
<img width="844" alt="image" src="https://github.com/woocommerce/woocommerce/assets/228780/e27f296d-22b9-45cf-8541-ff5de39f6247">

The alternative seems to work with both